properties/P000024.md

```diff
@@ -1,11 +1,14 @@
 ---
 uid: P000024
-slug: strongly-locally-compact
-name: Strongly Locally Compact
+slug: weakly-locally-closed-and-compact
+name: Weakly Locally Closed-and-Compact
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
+  - wiki: Locally_compact_space
+    name: Locally compact space
 ---
-A space $X$ is strongly locally compact if every point in $X$ has an open neighborhood with compact closure.
+A space $X$ is weakly locally closed-and-compact if every point in $X$ has
+a closed and compact neighborhood.
```

Comment:
I assume this is meant to be the same as the previous meaning of P000024, right?  If so, rephrasing as "locally relatively compact if every point in X has a relatively compact nbhd" would be better, I think.  Because that automatically implies there is a local base of relatively compact nbhds.  (that's the equivalence (2') = (2'') in wikipedia.

But saying "every point in X has a closed and compact nbhd" does not imply "there is a local base of closed and compact nbhds".  The latter would be the notion "local compactness via compact closures inside neighbourhoods" (def 2.2 of https://ncatlab.org/nlab/show/locally+compact+topological+space), which is not even listed in the wikipedia article.
(i.e., do we require the compact closure to be within a neighborhood or not)

Reply:
Since I'm editing P24 in-place I didn't intend to change its definition. I believe having an open nbhd with compact closure is equivalent to having a closed-and-compact neighborhood.

But as you point out, the name could be simplified as "locally relatively compact" due to (2')=(2'') in Wikipedia. Since that's less of a mouthful, it's probably better for the default name (we can have the current one as an alias).

Reply:
It's equivalent for the "weakly" version, but would not be equivalent for the "strongly" version (for the strongly version requiring an arbitrary nbhd to contain a closed-and-compact nbhd is not the same as requiring to contain a nbd with compact closure (as the closure could be outside the containing nbhd).

In any case, did you see my comment in issue #42 ?  I was playing around with some names.  See the last one in particular.  What do you think?